src/%IPM/Repo/Filesystem/Definition.cls

```diff
@@ -20,25 +20,6 @@ Property Depth As %Integer [ InitialExpression = 0, Required ];
 /// Prompt to use for Root in interactive configuration of this repository type
 Parameter RootPromptString = {$$$Text("Root File Path:","ZPM")};
 
-XData Commands
```

Comment:
This should not be removed. Needed to register additional options for repo command.